server/server.ts

```diff
@@ -141,8 +141,6 @@ function getNextTile(x: number, y: number, direction: Direction): { x: number; y
       return pixelToTile(x + PLAYER_SPEED, y);
     case Direction.None:
       return { x, y };
-    default:
-      return direction; //never,
```

Comment:
i'm a strong proponent of this line i think it's a good practice

Reply:
ok we can add it back